docs/whitepaper/federated_sidetree.md

```diff
@@ -274,5 +308,84 @@ The next step for this document is to list concrete representations and APIs. As
     * [ActivityPub](https://www.w3.org/TR/activitypub/) is a protocol in Fediverse implementations for push-based activity notifications.
-* Multitree registry implementation
-  * An additional project, leveraging the existing [sidetree-core-go](https://github.com/trustbloc/sidetree-core-go), is envisioned to support the multitree instance features.
+    * Potentially [GossipSub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub). This is an IPFS protocol for gossiping messages. However we want to be careful not to couple ourselves to IPFS networks.
+* Registry insance implementation
+  * An additional project, leveraging the existing [sidetree-core-go](https://github.com/trustbloc/sidetree-core-go), is envisioned to support the instance features.
+
+# Appendices
+
+## Moving between trees using existing Sidetree operations
+
+As a thought experiment, we could attempt to create rules to disambiguate suffix chain length without creating explicit operations. The suffix chain length is based on both the Sidetree recovery and update commitments as follows:
+
+* A suffix chain with a valid deactivate operation wins. The DID is deactivated.
+  * Note: This rule is present as the DID Controller showed their intent to deactivate in some known tree. However, it could be also be argued that the longest chain of recovery commitments always takes precedence.
+* The longest chain of Sidetree recovery commitments wins.
+* In case of a tie between two trees' suffix recovery chain length, the longest chain of update commitments wins.
+* In case of a tie between two trees' suffix chain length with the same operations (no conflict), there is no issue.
+* In the ambiguous case of a tie between two trees' suffix chain length with different final operations, the final ambiguous operation is ignored (and not applied). Note: to properly break the tie, the DID controller should apply an operation to one of the trees.
+
+When moving a DID to a new tree (without defining a new Sidetree operation):
+
+* The controller MUST publish the existing creation and recovery sequences onto the new tree. Note: the need to preserve the suffix chain may increase the burden on the DID controller.
+* The controller MUST publish a new recovery operation onto the new tree (and not the old tree) with the latest document content.
+* We may also wish to signal that operations will no longer be on the old tree. However, the existing Sidetree deactivate operation semantics do not completely match the semantics in this document (despite some similarity). An explicit export operation could be defined for this case.
+
+## Concrete registry instance design
+
+In this section, we present a design based on existing relevant technologies. Concrete design considerations for the Hubs is a followup task.
+
+### Technologies
+
+This concrete design leverages:
+
+* IPFS and IPLD.
+* Sidetree protocol (assuming extension operation for a signed variation of create, as a checkpoint).
```

Comment:
I see - there will be signed variation of create, as a checkpoint. You should probably mention all checkpoint variations in the "DID strings and resolution" section.

Reply:
It should be the other text. I'll fix.

Reply:
Fixed in https://github.com/trustbloc/trustbloc-did-method/pull/244